build/DependencyVersions.props

```diff
@@ -17,7 +17,7 @@
 
     <CLI_NuGet_Version>4.4.0-preview1-4434</CLI_NuGet_Version>
     <CLI_NETStandardLibraryNETFrameworkVersion>2.0.0-preview3-25514-04</CLI_NETStandardLibraryNETFrameworkVersion>
-    <CLI_WEBSDK_Version>2.0.0-rel-20170629-588</CLI_WEBSDK_Version>
+    <CLI_WEBSDK_Version>2.0.2-vspre-20170825-1</CLI_WEBSDK_Version>
```

Comment:
This is not right. You should be updating CLI_NETSDK_Version

Reply:
So only stage0, but the sdk used in produce will not be updated?

Reply:
No, it will update the product. This is updating a different component in the product. The version numbering scheme is different.